ros-ocp/values.yaml

```diff
@@ -121,6 +121,26 @@ minio:
     - koku-bucket
     - ros-data
 
+# ODF (Production Storage - for OpenShift deployments)
+odf:
+  # ODF S3 endpoint (will be auto-detected from ODF installation)
+  endpoint: ""
+  # ODF S3 credentials (configure these values for your ODF installation)
+  # Get access key: oc get secret noobaa-admin -n openshift-storage -o jsonpath='{.data.AWS_ACCESS_KEY_ID}' | base64 -d
+  # Get secret key: oc get secret noobaa-admin -n openshift-storage -o jsonpath='{.data.AWS_SECRET_ACCESS_KEY}' | base64 -d
+  accessKey: "OUOJaGX15HKZ7XJ0imdT"
+  secretKey: "LYOBsoS5iH8lpyweJDjXUYWOgLwDA8Z2q/HG8rgu"
```

Comment:
shouldn't this be somewhere outside the values.yaml? like in an existing secret that needs to be populated before the chart is created?